scripts/evaluate_similarity.py

```diff
@@ -18,18 +18,28 @@ def parse_args(args):
         help="Similarity algorithm to use",
         choices=["lsi", "neighbors_tfidf", "neighbors_tfidf_bigrams", "word2vec_wmd"],
     )
+    parser.add_argument(
+        "--disable-url-cleanup",
+        help="Don't cleanup urls when training the similarity model",
+        dest="cleanup_urls",
+        default=True,
+        action="store_false",
+    )
     return parser.parse_args(args)
 
 
 def main(args):
     if args.algorithm == "lsi":
```

Comment:
It's not great that we have to pass the same argument to the different constructors.
I'd do something like:
```
if args.algorithm == "lsi":
    model_creator = LSISimilarity
elif args.algorithm == "neighbors_tfidf":
   model_creator = NeighborsSimilarity
...

model = model_creator(cleanup_urls=args.cleanup_urls)
```